pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -116,6 +119,7 @@ def __call__(self, value, system):
             data=json.dumps(spec)
         )
         try:
+            log.debug('Validation of the TOC length with compute_toc_pages set to {} and default_toc_length set to {}'.format(print_config.get('compute_toc_pages'), print_config.get('default_toc_length'))) # noqa
             if Config.get('print', {}).get('compute_toc_pages', False):
```

Comment:
we can leave the if statement away.
The check if the toc pages from the actual pdf fits the extract_as_dict['nbTocPages'] we've set above should be done anyways and will be the same for all three cases.

Reply:
Agree.